packages/master-detail-layout/src/vaadin-master-detail-layout.d.ts

```diff
@@ -11,7 +11,37 @@ import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mix
  * (or primary) area and a detail (or secondary) area that is displayed next to, or
  * overlaid on top of, the master area, depending on configuration and viewport size.
  */
-declare class MasterDetailLayout extends ThemableMixin(ElementMixin(HTMLElement)) {}
+declare class MasterDetailLayout extends ThemableMixin(ElementMixin(HTMLElement)) {
```

Comment:
Is there a reason to implement `ThemableMixin` in new components, as shadow DOM styling is deprecated?

Reply:
I think I ended up implementing it in Card for some reason, but I didn't have a good reason (probably only because I copy-pasted from another component and couldn't make it work without it).

Reply:
Ah, right, it’s for our own needs. That was the reason in Card as well, that there isn't any other official way to load the theme stylesheets for our components.